app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,60 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
+
+import java.util.Objects;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+/**
+ *
+ */
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    private EditText username;
+
+    private EditText password;
+
+    private Button loginButton;
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        this.username = Objects.requireNonNull(getView()).findViewById(R.id.vLoginUsername);
```

Comment:
It's not really necessary to use this. We use `Butterknife` to bind views. Read a bit about it, don't hesitate to ask me.